cvpy/tests/test_image.py

```diff
@@ -24,6 +24,7 @@
 import numpy as np
 from cvpy.image import *
 
+
```

Comment:
Nit: any reason this line was added?

Reply:
No real reason - I think I added it to appease the PyCharm Linter. I will reach out separately about preferred Python style!